src/control/Control.Layers.js

```diff
@@ -85,11 +85,15 @@ export const Layers = Control.extend({
 		this._handlingClick = false;
 
 		for (const i in baseLayers) {
-			this._addLayer(baseLayers[i], i);
+			if (Object.hasOwn(baseLayers, i)) {
+				this._addLayer(baseLayers[i], i);
+			}
 		}
 
 		for (const i in overlays) {

```

Comment:
Seme here, I think we can reduce the complexity of this code by using `Object.entries()`.

Reply:
Similarly, the viability of this change hinges on whether baseLayers in guaranteed to be not null/undefined.